myMath.js

```diff
@@ -1,13 +1,25 @@
 'use strict';
 
+import mathjs from 'mathjs';
+
 class MyMath {
-    /**
-     * @param {string} stringValue
-     * @return {number}
-     */
-    static toInt(stringValue) {
-        return parseInt(stringValue, 10);
-    }
+  /**
+   * @param {string} stringValue
+   * @return {number}
+   */
+  static toInt(stringValue) {
+    return parseInt(stringValue, 10);
+  }
+
+  /**
+   * @param {number} number
+   * @return {number}
+   */
+  static sqrt(number) {
+    return mathjs.sqrt(number);
```

Comment:
`Math.sqrt` is a  native javascript function https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sqrt